package.json

```diff
@@ -20,7 +20,19 @@
         "async"
     ],
     "version": "1.3.2",
-    "main": "./src/index.ts",
+    "main": "./dist/fp-toolkit.cjs",
+    "module": "./dist/fp-toolkit.mjs",
+    "types": "./dist/fp-toolkit.d.ts",
+    "exports": {
+        ".": {
+            "require": "./dist/fp-toolkit.cjs",
+            "import": "./dist/fp-toolkit.mjs",
+            "types": "./dist/fp-toolkit.d.ts"
+        }
+    },
```

Comment:
Used https://antfu.me/posts/publish-esm-and-cjs as a reference for this. Will test shortly.

Reply:
Wow this is awesome. What a great find and thank you!

Reply:
> Will test shortly.

Does it blend??

Reply:
I got pulled away by build failures at work. 😅 Will test on Monday.

Reply:
I'm noticing the blog post mentions using this as a template for `tsup`. But it doesn't look like we installed the `tsup` package. Does vite do this for us?

Reply:
Right, tsup is just a transpiler / bundler. You already had the [vite config](https://github.com/fp-toolkit/fp-toolkit/blob/ed6c1c6853142fe800c5e01b6ec506230de7c413/vite.config.ts#L18-L20) set up to generate the bundles, we just weren't publishing them / exposing them here in the package.json.